pkg/pillar/cmd/zedagent/attesttask.go

```diff
@@ -360,16 +360,22 @@ func (server *VerifierImpl) SendAttestQuote(ctx *zattest.Context) error {
 		//Retrieve integrity token
 		storeIntegrityToken(quoteResp.GetIntegrityToken())
 		log.Notice("[ATTEST] Attestation successful, processing keys given by Controller")
+		publishedStorageKeys := 0
 		if encryptedKeys := quoteResp.GetKeys(); encryptedKeys != nil {
 			for _, sk := range encryptedKeys {
 				encryptedKeyType := sk.GetKeyType()
 				encryptedKey := sk.GetKey()
 				if encryptedKeyType == attest.AttestVolumeKeyType_ATTEST_VOLUME_KEY_TYPE_VSK {
 					publishEncryptedKeyFromController(attestCtx, encryptedKey)
 					log.Noticef("[ATTEST] published Controller-given encrypted key")
+					publishedStorageKeys++
 				}
 			}
 		}
+		if publishedStorageKeys == 0 {
+			publishEncryptedKeyFromController(attestCtx, nil)
+			log.Noticeln("[ATTEST] no keys received from controller, published dummy key")
```

Comment:
This should be a warning.

Reply:
I changed this part of code and modified verbosity.